src/hu_stuff.h

```diff
@@ -94,7 +94,15 @@ extern int playback_tic, playback_totaltics;
 extern int crispy_hud;
 
 extern int hud_crosshair;
-extern boolean hud_crosshair_health;
+
+typedef enum
+{
+  crosshealth_off,
+  crosshealth_player,
+  crosshealth_target,
+} crosshealth_t;
```

Comment:
Should we keep this type name? Personally, I want something like this; concise, not very long.